docs_src/microservices/security/edgex-secrets-config-proxy.md

```diff
@@ -0,0 +1,183 @@
+% edgex-secrets-config-proxy(1) User Manuals edgex-secrets-config-proxy(1)
+
+# NAME
+
+edgex-secrets-config-proxy – Configure EdgeX proxy service
+
+# SYNOPSIS
+
+**edgex-secrets-config proxy** [OPTIONS] SUBCOMMAND [ARG...]
```

Comment:
still missing a "-" between "config" and "proxy"?

Reply:
Ah, I was actually trying to plan for future functionality.

Proxy is just one tool that needs secrets configured and the most urgent right now.  But I think there may be a use case to configure secrets for other subsystems:

For example:
docker run
docker exec

And for this tool
edgex-secrets-config proxy
edgex-secrets-config appservice
edgex-secrets-config broker
etc

Reply:
But I can go with a single-function utility too.  Let me know what you prefer.